### PR TITLE
video mute/unmute button

### PR DESCRIPTION
**Pull request by intergalacticspacehighway**
# Why
Video lacks ability to mute/unmute audio

# How
add a mute/unmute button in video component

# Test Plan
Test video mute/unmute on all the platforms

# Aside
Follow up PR would be to experiment with FlashList and auto unmute videos based on user preference. I think we should try FlashList because viewability tracking with RecyclerListView on Android is not accurate and has some issues.

https://user-images.githubusercontent.com/23293248/178455250-c62a491c-6e59-4a91-b48d-ce5ee776bedd.mov

### CODE REVIEW

File: packages/app/components/mute-button/mute-button.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import { Platform, Pressable, StyleSheet } from "react-native";
+
+import { Muted, Unmuted } from "design-system/icon";
```
Comment (intergalacticspacehighway) [inline packages/app/components/mute-button/mute-button.tsx]: replacing it with showtime universal imports

**Review by axeldelafosse — APPROVED** [review]
LFG ✨

**Review by alantoa — APPROVED** [review]
LGTM! ✨

**Comment by alantoa** [comment]
> I think we should try FlashList  

@intergalacticspacehighway  cool! 
I'll replace `TabView`'s `RecyclerListView` with `FlashList` when you have done the feed!

**Comment by alantoa** [comment]
btw do we need to use `FlashList` on the web?
tbh on the web, I really don't like how it behaves, I would prefer a virtual list.

**Comment by intergalacticspacehighway** [comment]
@alantoa i'll try it today and let you know the findings!

**Pull request merged**